app/javascript/components/applications/application_form/preferences/PreferencesSection.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react'
-import { forEach } from 'lodash'
+import { forEach, isEmpty } from 'lodash'
 import PreferenceForm from './PreferenceForm'
 import { naturalKeyFromPreference, getFullHousehold } from './utils'
 
 const allPreferencesSelected = (formApi, listingPreferences) => {
-  if (formApi.values && formApi.values.preferences && listingPreferences) {
+  if (formApi.values && !isEmpty(formApi.values.preferences) && listingPreferences) {
     return formApi.values.preferences.length === listingPreferences.length
   }
 }
 
 const hasHouseholdMembers = (formApi) => {
-  let hasPrimaryApplicant = formApi.values.applicant && formApi.values.applicant.first_name
-  let hasHouseholdMembers = formApi.values.household_members && formApi.values.household_members.length
+  let hasPrimaryApplicant = !isEmpty(formApi.values) && formApi.values.applicant.first_name
```

Comment:
@akegan is there are reason we don't check for the presence of `formApi.values.applicant` here?

Reply:
no I'm not sure. This was a chris update. I'm not 100% sure where this is being used.

Reply:
ok, i'm going to add a `!isEmpty(formApi.values.applicant)` condition here to be safe